libr/include/r_util/r_assert.h

```diff
@@ -78,28 +76,28 @@ R_API void r_assert_log(RLogLevel level, const char *fmt, ...);
 #define r_return_if_fail(expr) \
 	do { \
 		if (!(expr)) { \
-			H_LOG_ (R_LOG_WARNING, "%s: assertion '%s' failed (line %d)\n", R_FUNCTION, #expr, __LINE__); \
+			R_LOG_WARN ("%s: assertion '%s' failed (line %d)\n", R_FUNCTION, #expr, __LINE__); \
```

Comment:
this should remain `H_LOG_`

Reply:
and all the others too

Reply:
Ah didn't realize I changed the H_LOG_ define as well as removed all usage of it! Thanks.